client-examples/BrutalmazeClient.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Net;
+using System.Net.Sockets;
+using System.Diagnostics;
```

Comment:
Wow these look so similiar to Python code (let's import the kitchen sink). Though still I need to ask you to check on this to make sure there isn't any unused one (damn I wish C# support namespace on library or sth, or does it?).